packages/block-library/src/media-text/edit.js

```diff
@@ -187,20 +187,6 @@ function MediaTextEdit( { attributes, isSelected, setAttributes } ) {
 		gridTemplateColumns,
 		msGridColumns: gridTemplateColumns,
 	};
-	const toolbarControls = [
```

Comment:
Should we discourage the usage of `controls` prop at the end of the refactoring with all non-default groups in `ToolbarGroup`?

Reply:
I think it's not supported in non default group blocks (I did this on purpose as I think component composition is better)